MultiCursor/MultiCursorModel.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "Engine/Node/CommonWidgets.hpp"
+#include <halp/audio.hpp>
+#include <halp/controls.hpp>
+#include <halp/meta.hpp>
+
+
+namespace Example
+{
+
+class MultiCursorManager
+{
+public:
+  halp_meta(name, "Multi-Cursor Manager")
+  halp_meta(category, "Control/Spatialization")
+  halp_meta(c_name, "multicursormanager")
+  halp_meta(uuid, "20f771a1-0e9a-4db2-bb7c-011467d84ded")
```

Comment:
also add yourself as the author, a description and add a documentation link (even if it does not exist yet, check the examples in avendish)

ex. from another object: 

```cpp
  halp_meta(name, "ADSR")
  halp_meta(c_name, "gamma_adsr")
  halp_meta(category, "Control/Mappings")
  halp_meta(description, "Trigger an ADSR envelope on input")
  halp_meta(manual_url, "https://ossia.io/score-docs/processes/adsr.html")
  halp_meta(author, "Lance Putnam, Gamma library")
  halp_meta(uuid, "2d603ea6-be84-4f8c-8269-643e989f5997")
  ```